### PR TITLE
Add Open Order QuoteResponse log

### PR DESCRIPTION
**Pull request by codyborn**
Add QuoteResponse logs for Open Orders

https://linear.app/uniswap/issue/PROTO-256/add-open-order-quoteresponse-log

### CODE REVIEW

File: lib/handlers/hard-quote/handler.ts [hunk 1]
```diff
@@ -87,6 +88,19 @@ export class QuoteHandler extends APIGLambdaHandler<
...
       log.info({ cosignerData: cosignerData }, 'open order with default cosignerData');
+
+      // The RFQ responses are logged in getBestQuote() 
+      // we log the Open Orders here
+      log.info({
+        eventType: 'QuoteResponse',
+        body: { 
+          ...logBody,
+          amountIn: request.totalInputAmountStart.toString(),
+          amountOut: request.totalOutputAmountStart.toString(),
```
Comment (codyborn) [inline lib/handlers/hard-quote/handler.ts:99]: @ConjunctiveNormalForm can you double check these should be the `amountIn` and `amountOut` values that we log for Open Orders?

File: lib/handlers/hard-quote/handler.ts [hunk 2]
```diff
@@ -51,25 +51,26 @@ export class QuoteHandler extends APIGLambdaHandler<
...
+      amount: request.amount.toString(),
+      type: TradeType[request.type],
+      numOutputs: request.numOutputs,
+      cosigner: request.order.info.cosigner,
+      createdAt: timestampInMstoSeconds(start),
+      createdAtMs: start.toString(),
+    }
     log.info({
       eventType: 'HardRequest',
-      body: {
```
Comment (ConjunctiveNormalForm) [inline lib/handlers/hard-quote/handler.ts:56]: What's the reason for moving this outside?

File: lib/handlers/hard-quote/handler.ts (hunk 1 above)
Comment (ConjunctiveNormalForm) [reply lib/handlers/hard-quote/handler.ts:99]: maybe better to use the response fields?

Comment (codyborn) [reply lib/handlers/hard-quote/handler.ts]: Great call out. I didn't see that.

File: lib/handlers/hard-quote/handler.ts (hunk 2 above)
Comment (codyborn) [reply lib/handlers/hard-quote/handler.ts]: It was to use the same object for logging the response. But no longer relevant with the latest changes.

**Review by ConjunctiveNormalForm — APPROVED** [review]
lgtm!

**Pull request merged**